### DISCUSSION TITLE
Multiple instances of Playtime

### DISCUSSION THREAD

**Discussion by RibbleFripFnibbulator** [helgoboss/helgobox#1271] [Q&A]
Hi, 

I've read some mention of using multiple Playtime instances in the same project. Is this currently possible to achieve?

**Comment by RibbleFripFnibbulator** [comment]
Ideally with an option to either sync tempo or have independent tempos...

**Comment by helgoboss** [comment]
Yes, it's possible to achieve. Just add multiple instances of Helgobox and show/hide the app using the Menu in the plug-in UI.

Tempos are always synced if in the same project. Why would you want different tempos?

**Reply by RibbleFripFnibbulator** [reply]
Ahh, "Show App"! Cool!

One example for different tempos would be to quickly resample something to another instance in its own tempo, clipping a little section spontaneously to create something weird or different, like an instant sample chop or stutter. Sometimes in the midst of performance I'm like "I wish I could grab this little section right now into its own 1 bar loop and do something else with it..." even if it had nothing in common with the tempo at which it was recorded. Like I would want to choose exactly where beat 1 is and where the bar should end. At the end of that recording, I would (usually) instantly stop all playback from "Deck A". Then I could build on the new capture in "Deck B", but still be able to return to the original performance at its original tempo, at the push of a button. 

I suppose I could fudge this together by storing the global tempo at the moment of the resample, and then setting it back to the stored value simultaneously with a "Stop Deck B / Start Deck A / restore original tempo" script. 

The ability to record unsynced loops alongside synced ones could also facilitate something like this (and other things). But the ability to have that new tempo become the "tempo master" is useful.
